File: wtplayer/src/WTCollisionInfo.cpp
#include "basetypes.hpp"
#include "WTCollisionInfo.hpp"
#include "WTObject.hpp"
#include "WTVector3D.hpp"

WTObject* WTCollisionInfo::getHitObject(){
  APILOG;
  return 0;
}

WTObject* WTCollisionInfo::getAttachedObject(){
  APILOG;
  return 0;
}

WTVector3D* WTCollisionInfo::getNewPosition(){
  APILOG;
  return 0;
}

WTVector3D* WTCollisionInfo::getImpactNormal(){
  APILOG;
  return 0;
}

float WTCollisionInfo::getImpactDistance(){
  APILOG;
  return 0;
}

File: wtplayer/javasrc/wildtangent/webdriver/WTCollisionInfo.java
package wildtangent.webdriver;

import wildtangent.webdriver.WTObject;
import wildtangent.webdriver.WTVector3D;

public interface WTCollisionInfo {
    public WTObject getAttachedObject();

    public WTVector3D getNewPosition();

    public float getImpactDistance();

    public WTVector3D getImpactNormal();

    public WTObject getHitObject();
}

